app/views/welcome/index.html.erb

```diff
@@ -1,3 +1,8 @@
+<%if user_signed_in?%>
+  <div class="position-absolute top-50 start-50 translate-middle">
+   <h1>Welcome to E-Market</h1>
```

Comment:
- [ ] Please make sure that your README file is professionally written and presented. Kindly fill in the empty sections with the correct information about the app besides fixing any broken links for a fully professional repo and README file. 😄  

_Please notice that the corresponding file for this requirement isn't loaded in your PR, so I had to leave the comment in a random file._

Reply:
fixed

Reply:
This issue is resolved.